trestle/core/commands/author/ssp.py

```diff
@@ -519,10 +563,18 @@ def _run(self, args: argparse.Namespace) -> int:
                     raise TrestleError('Original ssp has no system component.')
                 comp_dict[const.SSP_MAIN_COMP_NAME] = sys_comp
 
+                ssp_sys_imp_comps = ssp.system_implementation.components
```

Comment:
@vikas-agarwal76 Just wanted to call out this logic specifically.  This is really the only change to `ssp-assemble` that was made though it would only apply to a leveraging SSP. All other changes are processed conditionally when the `leveraged-ssp` flag is used (`generate`) or the inheritance markdown is present (`assemble`).